README.md

```diff
@@ -36,33 +36,57 @@ $ ansible-playbook -i localhost, --connection=local \
 
 ## Deploy new cluster + KubeVirt
 
-
 ### Requirements
+
+Make sure that you have Ansible 2.3.1.
+
 Install depending roles, and export `ANSIBLE_ROLES_PATH`
-```
+
+```bash
 $ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
 $ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
 ```
+
 For OpenShift deployment clone [**OpenShift Ansible project**](https://github.com/openshift/openshift-ansible)
-```
-$ git clone https://github.com/openshift/openshift-ansible
+
+```bash
+$ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
+### Parameters
+
+| Name             |  Value        | Description                            |
+| ---------------- | ------------- | -------------------------------------- |
+| cluster\_type    | `kubernetes`, `openshift` | Desired cluster            |
+| mode             | `release`, `dev` | If `dev` it will build KubeVirt from sources |
+| openshift\_ansible\_dir | string | Path to OpenShift Ansible repository   |
+
 ### Kubernetes
-Preparing Kubernetes cluster and deploy KubeVirt on it.
+
 - Add your master and nodes to `inventory` file.
-- Run ansible playbook `# ansible-playbook -i inventory deploy-kubernetes.yml`.
 
 ### OpenShift
-Preparing OpenShift cluster and deploy KubeVirt on it.
+
 - Be sure that you have enough space on your hosts for docker storage and
-edit openshift/roles/docker-setup/defaults/main.yaml accordingly.
-You can read more about docker-storage-setup [**here**](https://docs.openshift.org/1.5/install_config/install/host_preparation.html#configuring-docker-storage).
+you can modify [defaults values](docker-storage-setup-defaults) accordingly.
+Follow [docker-storage-setup] documentation for more details.
 - Add your master and nodes to `inventory` file.
-- Run ansible playbook `# ansible-playbook -i inventory -e "openshift_ansible_dir=... deploy-openshift.yml`.
-You must give directory where you placed `openshift-ansible` to the variable `openshift_ansible_dir`.
+
+
+### Example how to run playbook
+
+```bash
```

Comment:
Maybe change it to `install-kubevirt-on-openshift.yml` which doesn't require root permissions?

Reply:
This playbook already has a usage described in above section.
I am trying to update readme how to use whole deployments, because current readme doesn't reflect status of repository and since ppl are supposed to use it and their own roles for other components we need to align it.

Reply:
`control.yml` playbook will not actually work because we don't have any other `provider` except `lago` ...
what do you suggest to get out of this?
Something like following ?
```
- include: deploy-with-{{ provider }}.yml
  when: provider is defined
```

Reply:
Now I am getting back in loop to our previous problem:
We don't have playbook which takes user from N clean machines to running cluster with KubeVirt on top of it.

Reply:
A user can use `deploy-openshift.yml` for installing Openshift and then use `install-kubevirt-on-openshift.yml` 
for installing Openshift. I don't think that it's too much to ask from a user.